Nosenti/certificate-manager---Innocent
# Task6-KAN-37 Supplier Lookup

#### PR description
- This PR looks up the supplier to be added to the certificate form

#### Ticket / Jira Link
- [KAN-37](https://task-force-pro-dccs.atlassian.net/jira/software/projects/KAN/boards/1?assignee=712020%3A67b4d316-bd6f-4b16-a625-69afa0078a10&selectedIssue=KAN-37)

#### Checklist:
- [x] I have provided the screenshots to show it works

#### How To Test Changes
- Click on Search button on the certificate form
- Modal should popup to show suppliers. 
- Seach and select suppliers
- Upon selection, the supplier should be added to the certificate form

#### Screenshots:
![image](https://github.com/user-attachments/assets/65250e21-087b-480f-919d-81de03cc8899)
